packages/pigeon/example/app/windows/runner/messages.g.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-// Autogenerated from Pigeon, do not edit directly.
+// Autogenerated from Pigeon (v17.0.0), do not edit directly.
```

Comment:
FYI, Pigeon also updated other platforms' comment to add a Pigeon version. I reverted the other platforms' changes.

Reply:
(Probably we need to do a pass over this package's CONTRIBUTING.md with updated info.)

Reply:
Ah, should I update this document then?

https://github.com/flutter/packages/blob/fe6111ba54651ea793879070f90b8df9a10feb58/packages/pigeon/example/app/README.md?plain=1#L6-L9

FYI, `tool/generate.dart` and does not appear to work on my machine. I'll debug

https://github.com/flutter/packages/tree/main/packages/pigeon/example/app#pigeon_example_app

Hm, I did try `tool/generate.dart` but that hangs on my machine. I'll debug it...

Reply:
I have a draft to fix the hang here: https://github.com/flutter/packages/pull/6198